### ISSUE TITLE
Ignore node_modules

### ISSUE THREAD

**Issue by DirtyF** [11ty/giffleball#2]
```gherkin
Given If I follow the part 2 tutorial
When I run `eleventy` after I installed the `file-size` package
Then eleventy will process the `node_modules` folder
```
Output:

```sh
eleventy
Writing _site/index.html from ./index.liquid.
Writing _site/node_modules/file-size/readme/index.html from ./node_modules/file-size/readme.md.
Wrote 2 files in 0.26 seconds
```

### Fix

In order to ignore the `node_modules` foler,  add an `.eleventyignore` file with:

```
node_modules
```

Maybe it's something that should be mentioned in the post?

**Comment by zachleat** [comment]
Yeah, that's fair. We also ignore things in .gitignore, which is what usually catches those!

**Comment by zachleat** [comment]
This will be fixed in v0.2.14: See https://github.com/11ty/eleventy/issues/54
